include/docca/base-stage1.xsl

```diff
@@ -171,7 +171,7 @@
   </xsl:template>
 
           <xsl:template mode="overload-list" match="memberdef">
-            <xsl:apply-templates select="briefdescription[not(. = ../preceding-sibling::*/briefdescription)]"/>
+            <xsl:apply-templates select="briefdescription[not(. eq preceding::briefdescription[parent::memberdef][1])]"/>
```

Comment:
LOL That's it?!

Reply:
Yeah, although the `preceding` axis can be slow. I'm going to check in a better implementation shortly.